src/get-started/java-binding.md

```diff
@@ -914,18 +914,18 @@ exonum-java generate-template \
 
 <!-- TODO: add a link to Exonum docs for passwords passing -->
 
-**Note:** in the following examples we do not provide a password for private
-node keys protection by passing `--no-password` flag. It is strictly recommended
+**Note:** the following examples omit setting up a password for private
+node keys protection and pass a `--no-password` flag. It is strictly recommended
 __not__ to use this flag in real-world scenarios.
```

Comment:
production? A network setup for system testing *is* a real-world scenario, but does not require passwords.